src/css/index.css

```diff
@@ -0,0 +1,42 @@
+* {
+  color: #282828;
+}
+
+.background {
+  background-image: url('../assets/third_image.png');
+  height: 100vh;
+  width: 100vw;
+  margin: 0;
+  background-repeat: no-repeat !important;
+  background-position: center !important;
+  background-size: cover !important;
```

Comment:
Is it necessary to use the `!important` flag in this scenario?

Reply:
yes sir